lib/text/ttml_text_parser.js

```diff
@@ -1243,12 +1250,14 @@ shaka.text.TtmlTextParser.styleNs_ = 'http://www.w3.org/ns/ttml#styling';
 shaka.text.TtmlTextParser.styleEbuttsNs_ = 'urn:ebu:tt:style';
 
 /**
- * The namespace URL for SMPTE fields.
- * @const {string}
+ * The supported namespace URLs for SMPTE fields.
+ * @const Array.<{string}>
  * @private
  */
-shaka.text.TtmlTextParser.smpteNs_ =
-    'http://www.smpte-ra.org/schemas/2052-1/2010/smpte-tt';
+shaka.text.TtmlTextParser.smpteNs_ = [
```

Comment:
Would it be better to name it smpteNsList_ or something similar to indicate it's an array rather than a string?

Reply:
Done